app/views/view_data/departments/show.html.erb

```diff
@@ -21,38 +21,62 @@
   </div>
 
   <div class="grid-row">
-    <div class="column-two-thirds department-context">
-      <h1 class="heading-medium">Data about this organisation</h1>
-      <p class="lede">
-        Showing data for the year from  <%= time_tag @metrics.time_period.starts_on, @metrics.time_period.starts_on.to_formatted_s(:abbreviated_day_month_year) %> to <%= time_tag @metrics.time_period.ends_on, @metrics.time_period.ends_on.to_formatted_s(:abbreviated_day_month_year) %>
+    <div class="column-two-thirds detail-context">
+      <ul class="list list-bullet">
+        <li>
+          <%= render 'view_data/metrics/time_period_selector', time_period: @metrics.time_period %>
+        </li>
+        <li>
+          This data is aggregated from <a href="<%= view_data_department_metrics_path department_id: @department.natural_key,group_by: Metrics::GroupBy::Service %>"><%= pluralize(@department.services_count, 'service')%></a>
+        </li>
+      </ul>
+    </div>
+  </div>
+
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">Transactions received</h1>
+      <p>
+        These numbers tell you how much the organisation was used, and which channels were used the most.
+      </p>
+      <p>
+        <a href="">See a detailed description of how transactions received is defined.</a>
       </p>
+    </div>
+  </div>
+
+  <hr/>
 
-      <p class="lede">
-        Data is aggregated from <a href="<%= view_data_department_metrics_path department_id: @department.natural_key,group_by: Metrics::GroupBy::Service %>"><%= pluralize(@department.services_count, 'service')%></a> in <a href="<%= view_data_department_metrics_path department_id: @department.natural_key,group_by: Metrics::GroupBy::DeliveryOrganisation %>"><%= pluralize(@department.delivery_organisations_count, 'delivery organisation')%></a>
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">Transactions processed</h1>
```

Comment:
Some of these descriptions are duplicated across the 3 so could potentially be put into partials and rendered?

Reply:
👍